dataframes/04-reading-messy-data-into-dataframes.ipynb

```diff
@@ -0,0 +1,502 @@
+{
```

Comment:
<p>To make this pass our CI run, any cell that intentionally errors needs to be tagged with a <code>raises-exception </code> tag. See https://nbsphinx.readthedocs.io/en/0.4.3/allow-errors-per-cell.html for more information.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dask/dask-examples/pull/157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>